wurst/systems/core/Tribe.wurst

```diff
@@ -113,11 +113,19 @@ public class Tribe
     function makePlayersObservers()
         getMembers().forEach(member -> member.makeObserver())
 
+    function displayDefeated()
+        tribes.forEach() (Tribe t) ->
+            if t != this and t.getForce().containsPlayer(localPlayer)
+                printTimed("The " + this.name.color(SPECIAL_COLOR) + " tribe has been defeated", 7)
+
     function wasDefeated()
         makePlayersObservers()
         defeated = true
         if tribe.containsPlayer(GetLocalPlayer())
             print("You have been defeated.".color(HIGHLIGHT_COLOR))
+        // We don't want to display the tribe name w/o -tb on do we?
```

Comment:
Change this to a `TODO` for when tribes are always enabled, e.g. `TODO: Remove this once tribes always exists.`